cmd/submit.go

```diff
@@ -114,6 +121,23 @@ func runSubmit(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	return nil
 }
 
+func getExerciseSolutionFiles(baseDir string) ([]string, error) {
+	v := viper.New()
+	v.AddConfigPath(filepath.Join(baseDir, ".exercism"))
```

Comment:
Pretty sure it's now `.meta`, not `.exercism`.

Reply:
It's `.meta/config.json` in the tracks repositories, but gets downloaded to `.exercism/config.json` when the student downloads the exercise.

Reply:
It makes sense to have the `.meta` folder in the test runner because like Erik says there, it is a copy of the exercise from the repository and in the repository the folder is `.meta`.

But locally on the student's computer it is `.exercism`:

https://github.com/exercism/cli/blob/fbd552b334235d4537d55af45331107c3ad09c12/workspace/exercise_metadata.go#L15-L17

Reply:
Pretty sure this convo can be marked as resolved; I just don't have any permissions in this repo.